### DISCUSSION TITLE
Which theme is this?

### DISCUSSION THREAD

**Discussion by 9mm** [b0o/incline.nvim#48] [Questions and Answers]
I like it

![](https://user-images.githubusercontent.com/21299126/167235114-d562ea45-155c-4d82-aaf1-95abb56398b7.png)

**Comment by b0o** [comment]
This is a custom theme I made, I call it “lavi”. You can find it in my [nvim-conf repo](https://github.com/b0o/nvim-conf/blob/main/colors/lavi.vim). I also have [a version for Alacritty](https://github.com/b0o/alacritty-conf/blob/main/src/themes/b0o/lavi.yml).
